source/documentation/index.md

```diff
@@ -540,35 +594,35 @@ These journeys show examples of use:
 
 Version 1.1
 
-* Payments and Liabilities section added.
+- Payments and Liabilities section added.
 
 Version 1.2
 
-* Multiple businesses section added.
-* Construction Industry Scheme (CIS) section added.
+- Multiple businesses section added.
+- Construction Industry Scheme (CIS) section added.
 
 Version 1.3
 
-* Losses updated.
-* Additional income section added.
+- Losses updated.
+- Additional income section added.
 
 Version 1.4
 
-* Construction Industry Scheme (CIS) section updated.
+- Construction Industry Scheme (CIS) section updated.
 
 Version 1.5
 
-* Production approvals process for Making Tax Digital for Self Assessment.
-* Early draft MTD ITSA Regulations 2021 section added.
+- Production approvals process for Making Tax Digital for Self Assessment.
+- Early draft MTD ITSA Regulations 2021 section added.
 
 Version 1.6
 
-* Revised wording for Draft MTD ITSA Regulations 2021 page.
+- Revised wording for Draft MTD ITSA Regulations 2021 page.
 
 Version 1.7
-* Change of title to Income Tax (Making Tax Digital) end-to-end Service Guide
-* Inclusion of Income Tax (MTD) APIs
-* Inclusion of Tax Terminology
-* Addition of Foreign Property reference to Obligations
-* Replaced the term, ‘taxpayers’ with ‘customers’
-* Replaced references to SA Accounting Summary API with BSAS API 
+- Change of title to Income Tax (Making Tax Digital) end-to-end Service Guide
+- Inclusion of Income Tax (MTD) APIs
+- Inclusion of Tax Terminology
+- Addition of Foreign Property reference to Obligations
+- Replaced the term, ‘taxpayers’ with ‘customers’
+- Replaced references to SA Accounting Summary API with BSAS API 
```

Comment:
This part of the documentation still doesn't seem to be generating bullet points